cha-ching-app/src/CheckoutForm.js

```diff
@@ -1,36 +1,42 @@
-import { PaymentElement, useStripe, useElements } from "@stripe/react-stripe-js";
+import {
+  PaymentElement,
+  useStripe,
+  useElements,
+} from "@stripe/react-stripe-js";
 
 function CheckoutForm() {
-    const stripe = useStripe()
-    const elements = useElements()
+  const stripe = useStripe();
+  const elements = useElements();
 
-    const handleSubmit = async () => {
-        await stripe.confirmPayment({
-            elements,
-            confirmParams: {
-                return_url: "http://localhost:3000/success",
-            },
-        })
-        .then(data => console.log("What am I (data):", data))
-        .catch(err => console.log("Something went wrong confirming the payment:", err))
-    }
+  const handleSubmit = async () => {
+    await stripe
+      .confirmPayment({
+        elements,
+        confirmParams: {
+          return_url: "http://localhost:3000/success",
+        },
+      })
+      .then((data) => console.log("What am I (data):", data))
+      .catch((err) =>
+        console.log("Something went wrong confirming the payment:", err),
+      );
```

Comment:
Stripe ends up handling errors that happen as a result of confirmPayment so it ends up printing out any form error in the `.then` block. Could we check that when we pass an undefined value inside the confirmPayment that it hits this catch block?